src/tests/build.sh

```diff
@@ -382,9 +381,11 @@ if [[ -z "$HOME" ]]; then
 fi
 
 if [[ "$__RebuildTests" -ne 0 ]]; then
-    if [[ -d "${__TestWorkingDir}" ]]; then
-        echo "Removing tests build dir: ${__TestWorkingDir}"
-        rm -rf "${__TestWorkingDir}"
+    if [[ -d "${__TestBinDir}" ]]; then
+        echo "Removing test build dir: ${__TestBinDir}"
+        rm -rf "${__TestBinDir}"
+        echo "Removing test intermediate dir: ${__TestIntermediatesDir}"
```

Comment:
Does this need a preceding:
```
if [[ -d "${__TestIntermediatesDir}" ]]; then
```
? Or will `rm -rf` not complain if it doesn't exist? Of course, if __TestBinDir exists, it will essentially always exist anyway.

Reply:
Thanks Bruce for pointing that out. I have locally verified that rm -rf has no trouble with non-existent folders so I just removed the pre-existing condition instead of introducing a new one.